src/configuration.ts

```diff
@@ -614,6 +631,24 @@ export function resolveTeamIds(cfg: Configuration): string[] {
     return cfg.teamIds;
 }
 
+/**
+ * Examine environment, config, and cfg for Atomist workspace IDs.  The
+ * ATOMIST_WORKSPACES environment variable takes precedence over the
+ * ATOMIST_WORKSPACE environment variable, which takes precedence over the
+ * configuration "workspaceIds", which takes precedence over cfg.workspaceId,
+ * which may be undefined, null, or an empty array.
+ */
+export function resolveWorkspaceIds(cfg: Configuration): string[] {
+    if (process.env.ATOMIST_WORKSPACES) {
+        cfg.workspaceIds = process.env.ATOMIST_WORKSPACES.split(",");
+    } else if (process.env.ATOMIST_WORKSPACE) {
```

Comment:
Can we take this opportunity to remove the singular version? I think it is more confusing to support two environment variables than it is to explain that the one we do support can accept a comma-delimited list of IDs.

Reply:
Will do